### ISSUE TITLE
Address Deprecation of Gradle OSGI Plugin

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-web#86]
Address Deprecation of Gradle OSGI Plugin
```
/data/xtext-master/git/xtext-web$ ./gradlew clean build -PuseJenkinsSnapshots=true --warning-mode all

> Configure project :
The osgi plugin has been deprecated. This is scheduled to be removed in Gradle 6.0. Consider using the biz.aQute.bnd plugin instead.
        at manifest_gen_6vfzhhhsie8v8m1761v0603i0.run(/data/xtext-master/git/xtext-web/gradle/manifest-gen.gradle:6)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
```

**Comment by cdietrich** [comment]
TODO:

investiage if we can do the same by normal jar manifest task

**Comment by holgerschill** [comment]
Fixed in 2.18
